packages/webapp/client/js/core.js

```diff
@@ -14,3 +27,80 @@ const jsElements = document.getElementsByClassName('defra-js-show')
 Array.prototype.forEach.call(jsElements, function (element) {
   element.removeAttribute('hidden')
 })
+
+const acceptButton = document.querySelector('.js-cookies-button-accept')
+const rejectButton = document.querySelector('.js-cookies-button-reject')
+const acceptedBanner = document.querySelector('.js-cookies-accepted')
+const rejectedBanner = document.querySelector('.js-cookies-rejected')
+const questionBanner = document.querySelector('.js-question-banner')
+const cookieBanner = document.querySelector('.js-cookies-banner')
+const cookieBannerContainer = document.querySelector('.js-cookie-banner-container')
+const cookiePrefsPage = document.querySelector('.defra-js')
+const cookiePageBanner = document.querySelector('.js-cookie-page-banner')
+const cookieSeenBanner = 'seen_cookie_message'
+const cookieSeenBannerExpiry = 30
+const cookieUserPreference = 'set_cookie_usage'
+const cookieUserPreferenceExpiry = 30
+
+const savePreference = accepted => {
+  const prefs = {
+    analytics: accepted ? 'on' : 'off'
+  }
+  window.bng.utils.setCookie(cookieUserPreference, JSON.stringify(prefs), cookieUserPreferenceExpiry)
+  cookiePageBanner.style.display = 'block'
```

Comment:
Accepting or rejecting on banner is throwing a JS error on setting this style.display, cookiePageBanner is null

Reply:
Presumably it only needs to execute if on cookies page, giving more weight to needing a cookies.js script file